.github/ISSUE_TEMPLATE/OSPO_Training_Request.md

```diff
@@ -0,0 +1,29 @@
+---
+name: OSPO Training Request Ticket
+about: Ticket for requesting OSPO Training
+title: Request OSPO Training
+labels: helpdesk
+assignees: ''
+
+---
+# Request OSPO Training
+
+## What is your name? (e.g. Firstname Lastname)
+
+- NAME
+
+## What is your email address? (e.g. - first.last@cms.hhs.gov)
+
+- EMAIL
+
+## What is your Component/Group/Division/Team? (e.g. OA/DSAC/OSPO or OIT/EADG/DEIMS/IDR)
+
+- CMP/GRP/DIV/TEAM
+
+## Which OSPO training course are you requesting? (E.g., - OSPO Overview, or GitHub Markdown Language, etc…)
```

Comment:
Not sure if we have this information written but if we do, could we add descriptions for each of the training courses we provide?